com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -1181,7 +1206,7 @@ bool CanCopyDepth(ref CameraData cameraData)
             bool msaaDepthResolve = msaaEnabledForCamera && SystemInfo.supportsMultisampledTextures != 0;
 
             // copying depth on GLES3 is giving invalid results. Needs investigation (Fogbugz issue 1339401)
```

Comment:
This issue was closed by design. Should we remove the comment "Needs investigation"?

Reply:
Maybe "needs driver fix"?

Reply:
changed it here:
https://github.com/Unity-Technologies/Graphics/pull/6217/commits/296526b63828a0529bfa703451730b99b54a50c3